recipes/universal-ctags/build.sh

```diff
@@ -0,0 +1,4 @@
+./autogen.sh
+./configure --prefix=$PREFIX
+make -j${CPU_COUNT}
+make install
```

Comment:
I'm assuming there is not a make check or similar, right?

Reply:
Maybe, I'll have a look on the feedstock